beacon-chain/forkchoice/doubly-linked-tree/node.go

```diff
@@ -9,8 +9,17 @@ import (
 	"github.com/prysmaticlabs/prysm/v3/config/params"
 	"github.com/prysmaticlabs/prysm/v3/consensus-types/primitives"
 	v1 "github.com/prysmaticlabs/prysm/v3/proto/eth/v1"
+	"github.com/prysmaticlabs/prysm/v3/time/slots"
 )
 
+// orphanLateBlockFirstThreshold is the number of seconds after which we
+// consider a block to be late, and thus a candidate to being reorged.
+const orphanLateBlockFirstThreshold = 4
+
+// processAttestationsThreshold  is the number of seconds after which we
+// process attestations for the current slot
+const processAttestationsThreshold = 10
```

Comment:
any thoughts on putting these as flags with default to 4 / 10 or we simply do not want ppl to change these

Reply:
I rather users not mess with these since that can cause more trouble than what we are trying to fix.